File: src/main/java/com/back/mozu/domain/setting/dto/SettingUpdateRequestDto.java
package com.back.mozu.domain.setting.dto;


import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

import java.time.LocalTime;

@Getter
@NoArgsConstructor
@AllArgsConstructor
public class SettingUpdateRequestDto {
    private Integer totalTables;
    private LocalTime openingTime;
    private LocalTime closingTime;
}

File: src/main/java/com/back/mozu/domain/setting/dto/SettingUpdateRequestDto.kt
package com.back.mozu.domain.setting.dto

import java.time.LocalTime

// PATCH 요청이라 일부 필드만 올 수 있어서 nullable 유지
// data class = 자바의 record
data class SettingUpdateRequestDto(
    val totalTables: Int?,
    val openingTime: LocalTime?,
    val closingTime: LocalTime?
)